nutkit/protocol/feature.py

```diff
@@ -88,6 +88,8 @@ class Feature(Enum):
     API_TYPE_SPATIAL = "Feature:API:Type.Spatial"
     # The driver supports sending and receiving temporal data types.
     API_TYPE_TEMPORAL = "Feature:API:Type.Temporal"
+    # The driver supports sending and receiving the unsupported type data type.
+    API_TYPE_UNSUPPORTED_TYPE = "Feature:API:Type.UnsupportedType"
```

Comment:
Isn't this part of Bolt 6.0? Maybe we don't need a feature flag for it.

Reply:
As far as I am aware, supporting this is not optional.

Reply:
I know. Personally, I would expect the driver to support both vectors and unsupported type so long as it claims support for Bolt 6.0.